src/osdc/Objecter.cc

```diff
@@ -2644,6 +2644,10 @@ int Objecter::_calc_target(op_target_t *t, epoch_t *last_force_resend,
     need_check_tiering = true;
   }
 
+  // Ignore tiering if doing scrub read
+  if (t->use_osd_epoch)
+    need_check_tiering = false;
```

Comment:
can we just add the `CEPH_OSD_FLAG_IGNORE_OVERLAY` bit in `operate_repair_read()`, or check `CEPH_OSD_FLAG_REPAIR_READS` here instead?